src/ImageSharp/Image/ImageBase{TPixel}.cs

```diff
@@ -37,6 +38,11 @@ public abstract class ImageBase<TPixel> : IImageBase<TPixel>
         private TPixel[] pixelBuffer;
 
         /// <summary>
+        /// The span representing the pixel buffer
+        /// </summary>
+        private Span<TPixel> span;
```

Comment:
I'm assuming I'm able to assign that once vs evaluating it on each request?

e.g.

```
this.pixelBuffer = newPixels;
this.Pixels = new Span<TPixel>(this.pixelBuffer, 0, this.Width * this.Height);
```

vs

```
public Span<TPixel> Pixels => new Span<TPixel>(this.pixelBuffer, 0, this.Width * this.Height)
```